zkvm/src/lib.rs

```diff
@@ -287,7 +251,7 @@ mod tests {
             .zip(&bi_files)
             .for_each(|(data, filename)| {
                 let mut f = fs::File::create(filename).unwrap();
-                for d in data {
+                for d in &data.0 {
```

Comment:
`riscv_continuations_dry_run` now returns a tuple of 2 things, here we're only interested in the bootloader values which is the first element of the tuple